app/models/user.rb

```diff
@@ -1,3 +1,15 @@
 class User < ApplicationRecord
   has_secure_password
+
+  before_create :set_auth_token
```

Comment:
let's remove this before_commit hook as we don't want to set this all the time we want to update it on each login